docs/products/kafka/reference/kafka-metrics-prometheus.md

```diff
@@ -144,6 +151,49 @@ health of your Apache Kafka controller.
 | `kafka_controller_KafkaController_TopicsIneligibleToDeleteCount_Value`                         | Number of topics ineligible to delete                                                       |
 | `kafka_controller_KafkaController_TopicsToDeleteCount_Value`                                   | Number of topics to delete                                                                  |
 
+#### KRaft mode and metrics changes
+
+Aiven for Apache Kafka services running Apache Kafka 3.9 operate in KRaft mode, which
+no longer exposes certain controller-related metrics previously available in
```

Comment:
We could be a little more specific here. The metrics are available in KRaft controller, but Aiven deliberately does not expose those, because we consider those service internal metrics, not something the end user needs to know.

Reply:
I have updated the content here. Once we have the other content KRaft, will cross-link for more details.